lambda__wzdx_ingest_to_archive.py

```diff
@@ -0,0 +1,32 @@
+"""
+Ingest raw data from WZDx feed to Work Zone Raw Data Sandbox.
+
+"""
+from __future__ import print_function
+
+import json
+import logging
+import os
+import traceback
+
+from wzdx_sandbox import WorkZoneRawSandbox
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)  # necessary to make sure aws is logging
+
+BUCKET = os.environ['BUCKET']
+LAMBDA_TO_TRIGGER = os.environ['LAMBDA_TO_TRIGGER']
+
+
+def lambda_handler(event=None, context=None):
+    """AWS Lambda handler. """
+    wzdx_sandbox = WorkZoneRawSandbox(feed=event['feed'], bucket=BUCKET,
+                    lambda_to_trigger=LAMBDA_TO_TRIGGER, logger=logger)
+    wzdx_sandbox.ingest()
+
+    logger.info('Processed events')
```

Comment:
Importing the print_function but using the logger here. 

Also, is there anything else that might be useful to log to improve traceability?

Reply:
Removed this log statement and added additional log statement in the wzdx sandbox classes.